tests/integration/targets/ansible_galaxy_install/files/test.yml

```diff
@@ -0,0 +1,11 @@
+---
+roles:
+  # Install a role from Ansible Galaxy.
+  - name: geerlingguy.java
+    version: 1.9.6
+
+collections:
+  # Install a collection from Ansible Galaxy.
+  - name: geerlingguy.php_roles
```

Comment:
CC @geerlingguy so you know that CI for this module will bump your download numbers. I hope that's fine for you :)

Reply:
@geerlingguy @felixfontein if there is any other collection and role that I could use instead, by all means just let me know.

Reply:
I think it's ok to use these ones, as long as Jeff doesn't veto. I would interpret no reaction as no veto, so it should be fine :)